### PR TITLE
New feature: Jump directly to a knot. 

### PR DESCRIPTION
**Pull request by cidneyhamilton**
Supports the following syntax in .esc files to jump directly to a knot in the script.

`play_ink res://path/to/ink/script.ink.json knot_name` 

If the second parameter is blank, the script will start from the top.

There's a new button in the test scene to test it out.

### CODE REVIEW

File: addons/escoria-ink-bridge/api/escoria_ink_bridge.gd [hunk 1]
```diff
@@ -33,7 +33,7 @@ func is_story_valid(ink_file: String) -> bool:
 	return not _get_story(ink_file) == null
 
 
-func run_story(ink_file: String):
+func run_story(ink_file: String, knot: String):
```
Comment (dploeger) [inline addons/escoria-ink-bridge/api/escoria_ink_bridge.gd]: Add a default value (, knot := "") for knot here to make it backwards compatible.

File: script_templates/ink_template.gd
Comment (dploeger) [inline script_templates/ink_template.gd:1]: Do you know where those script templates come from? Are they generated by inkgd? Maybe we can get rid of them in the repository.

**Review by dploeger — CHANGES_REQUESTED** [review]
Some small remarks

File: script_templates/ink_template.gd
Comment (cidneyhamilton) [reply script_templates/ink_template.gd:1]: Adding to .gitignore to keep them out of the repository.

**Comment by cidneyhamilton** [comment]
Fixes in (and new commits squashed).

File: script_templates/ink_template.gd
Comment (cidneyhamilton) [reply script_templates/ink_template.gd:1]: (and yep, it's all been generated.) I'm erring on the side of not making changes unless specified.

**Review by dploeger — APPROVED** [review]
Awesome! Thanks a lot. 🎁

**Pull request merged**